stable/mlrun/values.yaml

```diff
@@ -240,6 +240,52 @@ api:
       periodSeconds: 15
       failureThreshold: 4
 
+  sidecars:
+    logCollector:
+      enabled: false
+      name: log-collector
+      fullnameOverride:
+
+      logLevel: debug
+      logFormat: text
+      listenPort: 8282
+
+      workdir: /var/mlrun/
+
+      image:
+        repository: mlrun/log-collector
+        tag: 1.3.0
+        pullPolicy: IfNotPresent
+        pullSecrets: []
+
+      extraEnv: []
+
+      extraEnvKeyValue: {}
+
+      envFrom:
+        - configMapRef:
+            name: mlrun-log-collector-config
+
+      securityContext: {}
+
+      resources:
+
+      readinessProbe:
+        exec:
+          command: ["/bin/grpc_health_probe", "-addr=:8080"]
+        initialDelaySeconds: 10
+        timeoutSeconds: 10
+        periodSeconds: 15
+        failureThreshold: 4
+
+      livenessProbe:
+        exec:
+          command: ["/bin/grpc_health_probe", "-addr=:8080"]
```

Comment:
I think for k8s >= 1.23 we can use https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe and avoice infamous liveness probe using exec